Library/Formula/openssl.rb

```diff
@@ -16,6 +16,7 @@ class Openssl < Formula
 
   option :universal
   option "without-check", "Skip build-time tests (not recommended)"
+  option "with-trace", "Build with enable-ssl-trace"
```

Comment:
Would you mind using a more understandable text for people who are not familiar with the `./configure` options? Something like “Enable support for the -trace option on s_client” for example.